proposals/BVAL-208.adoc

```diff
@@ -0,0 +1,142 @@
+—
+title: Support groups translation when cascading operations
```

Comment:
[BVAL-208](https://hibernate.atlassian.net/projects/BVAL/issues/BVAL-208) has been addressed in the BV 1.1 timeframe already. I probably wouldn't have gone for updating old proposals, even would say we can delete them (the will still be available in Git for historical purposes, if ever needed).